langchain/utils.py

```diff
@@ -111,3 +111,14 @@ def now(cls):  # type: ignore
         yield datetime.datetime
     finally:
         datetime.datetime = real_datetime
+
+
+def truncate_word(content: str, length: int = 300, suffix: str = "...") -> str:
```

Comment:
Could we introduce the star operator, it makes it easier to enforce good calling etiquette for interfaces with named arguments being invoked as named arguments -- this tends to make caller code more readable and makes it easier to do certain kinds of refactors without introducing breaking changes

```suggestion
def truncate_word(content: str, *, length: int = 300, suffix: str = "...") -> str:
```

Reply:
@wsantos apologies edited the above ^ a few times post fact